internal/pkg/levee/testdata/src/levee_analysistest/example/tests/stdlib/funcs.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stdlib
```

Comment:
This test collection does a good job testing the positive case, but could use some negative-cases.

In that scope, many follow an `allocate, taint, sink, //want` pattern.  Adding the negative case prior to taint, i.e., `allocate sink-but-no-want, taint, sink, //want`, could be good for a wider confirmation that we aren't propagating "backwards."

Reply:
I felt like backwards propagation was already sufficiently covered by existing tests in e.g. `callorder`, but it can't hurt to have a few negative cases.

I will add those soon.

Reply:
I've tweaked a case to sink a `Writer` before it gets tainted. IMO this is sufficient since flow sensitivity is already well covered by existing cases (mostly `callorder`).

I think there is also adequate coverage for other kinds of negative cases, e.g. function returns 2 values, only one of which is tainted.

Reply:
LGTM